packages/react/src/hooks/useRCAuth.js

```diff
@@ -56,6 +60,7 @@ export const useRCAuth = () => {
           setIsTotpModalOpen(false);
           setEmailorUser(null);
           setPassword(null);
+          setUserPinPermissions(permissions.update[150]);
```

Comment:
Hey @SinghaAnirban005, I think it is permissions.update[146] according to the previous pr - #520

Reply:
The 150th index returns the pin permissions correctly

Reply:
okay then what is on 146th??

Reply:
check and try pr 520

Reply:
i am getting this log on 146 
![Screenshot 2024-12-17 004227](https://github.com/user-attachments/assets/a0b7a9a6-cc77-410e-aeaf-580597c5bdef)

Reply:
Ohh so might this permissions array got changed later

Reply:
Yes exactly